homeassistant/setup.py

```diff
@@ -163,8 +163,9 @@ def log_error(msg, link=True):
         loader.set_component(hass, domain, None)
         return False
 
-    for entry in hass.config_entries.async_entries(domain):
-        await entry.async_setup(hass, component=component)
+    if hass.config_entries:
```

Comment:
Why this if check?

Reply:
`hass.config_entries` is initially `None`.